crates/consensus/src/transaction/deposit.rs

```diff
@@ -241,6 +243,28 @@ impl Decodable for TxDeposit {
     }
 }
 
+impl SignableTransaction<alloy_primitives::Signature> for TxDeposit {
```

Comment:
```suggestion
impl SignableTransaction<Signature> for TxDeposit {
```
nitpick, bring this into scope in module import prelude